lib/brainstem/dsl/field.rb

```diff
@@ -51,6 +53,27 @@ def conditionals_match?(model, presenter_conditionals, helper_instance = Object.
           presenter_conditionals[conditional].matches?(model, helper_instance, conditional_cache)
         }
       end
+
+      private
+
+      def run_on_with_lookup(model, context, helper_instance)
```

Comment:
This is addressed here: https://github.com/mavenlink/brainstem/pull/79/commits/712b392d86e2958ce8cc63e519a945f97fe6959d